cmd/updater/main.go

```diff
@@ -375,6 +375,7 @@ func AppendColumn(headers []string, format nameConfig, grid *state.Grid, rows ma
 				r = &state.Row{
 					Name: name,
 					Id:   target,
+					// TODO(fejta): AlertInfo,
```

Comment:
I think I see some issues with how we open/close these alerts, but I eventually realized the updater is what is responsible for this, not the summarizer. So any issues are not really relevant here, added a TODO to move this logic into go